{"name": "Проверка тело API на ошибку логинизации при неправильном пароле", "status": "passed", "start": 1712696808944, "stop": 1712696809001, "uuid": "f5a06cc8-a4d7-4336-b895-1593d49814ad", "historyId": "b0fb7c981c96245c0fe0bd4d68a4ea3d", "testCaseId": "b0fb7c981c96245c0fe0bd4d68a4ea3d", "fullName": "tests.test_login.TestLogin#test_wrong_login_password_body", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "maksimka"}, {"name": "thread", "value": "49960-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}